services/src/main/java/io/druid/cli/CliBroker.java

```diff
@@ -102,6 +101,7 @@ public void configure(Binder binder)
 
             binder.bind(JettyServerInitializer.class).to(QueryJettyServerInitializer.class).in(LazySingleton.class);
             Jerseys.addResource(binder, QueryResource.class);
+            Jerseys.addResource(binder, QueryResourceV3.class);
```

Comment:
@nishantmonu51 this PR is for only enabling the endpoint on broker.

full historical and realtime node support is being added in #3323 

i have to admit that I had forgotten about the router, adding router in #3323 as well. (i guess router will be simpler than historical/realtime because router does not deserialize the response and just a pass through )